src/main/java/org/embulk/output/mailchimp/MailChimpRecordBuffer.java

```diff
@@ -210,17 +208,10 @@ public JsonNode apply(JsonNode input)
                         if (!"".equals(containsCaseInsensitive(column.getName(), task.getMergeFields().get()))) {
                             String value = input.hasNonNull(column.getName()) ? input.findValue(column.getName()).asText() : "";
 
-                            // Try to convert to Json from string with the merge field's type is address
-                            if (availableMergeFields.get(column.getName().toLowerCase()).getType()
+                            if (availableMergeFields.get(column.getName().toLowerCase()) != null && availableMergeFields.get(column.getName().toLowerCase()).getType()
```

Comment:
should we use ` availableMergeFields.has(column.getName().toLowerCase())` instead of?

Reply:
Look shorter . Updated 👍